### DISCUSSION TITLE
Serializing Date Objects Causes Log Not To Emit

### DISCUSSION THREAD

**Discussion by asafpelegcodes** [logfellow/logstash-logback-encoder#989] [Q&A] [answered]
Settings...
Logstash-logback encoder version: 7.4 
Kotlin, Ktor Server
io.github.oshai.kotlinlogging.KotlinLogging
org.joda.time.DateTime

When I log datetime objects using Kotlin Logging and drop them into marker KV properties the log simply does not append to the console and appears to silently fail.

```
import io.github.oshai.kotlinlogging.KotlinLogging
import org.joda.time.DateTime

private val logger = KotlinLogging.logger {}

fun foo() {
  logger.atInfo {
        message = "My message"
        payload = buildMap {
          put("dateKey", DateTime.now().toLocalDate())
        }
      }
}
```

**Comment by philsttr** [comment, marked as answer]
Errors that occur within logback during encoding/appending can be seen by enabling a logback [status listener](https://logback.qos.ch/manual/configuration.html#dumpingStatusData). This is a logback feature, and not specific to logstash-logback-encoder.

Try enabling the status listener and see if any errors are reported.
